testing/jenkins-kube-creds.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: jenkins-k8s-creds
+  name: jenkins-k8s-creds
+  namespace: testing
+spec:
+  replicas: 1
+  template: 
+    metadata:
+      labels:
+        app: jenkins-k8s-creds
+    spec:
+      containers:
+      - name: jenkins-k8s-creds
+        imagePullPolicy: Always
```

Comment:
FYI, you can use `imagePullPolicy: Never` if you are running on a local kubernetes cluster (ie, on docker on your laptop). That way, it will never try to pull the image and so will get it from the local cache, which will be either the latest cache that you pulled, or (more usefully) the latest version that you built.

Reply:
Because I am using GKE for testing and using the "latest" I need the Always...